selftests/check.py

```diff
@@ -518,6 +518,13 @@ def create_suites(args):
     return suites
 
 
+def print_failed_tests(tests):
+    if tests:
```

Comment:
`get_failed_tests()` is returning always a list. So looks like, this `if` is unnecessary.

Reply:
Yes, sure

Reply:
Oh makes sense, I missed that.